ITISFoundation/osparc-simcore
# Some enhancements and bug fixes

## What do these changes do?


* Fixes wrong subject in service submission email
* **Added osparc.window.Window component with ``clickAwayClose`` function.**
* Added faded modal overlay.

## Related issue number


closes #1567 #1609 #1607 

## How to test


* Service submission email has right subject
* Check window styles, modal overlays and click away close
* Can edit study info from dashboard
* Dashboard studies update after name change inside study

## Screenshot
#### Click away to close and modal overlay
In PR, click away is only activated for permissions window.
![clickaway](https://user-images.githubusercontent.com/4764217/86917942-3cba9400-c126-11ea-84f5-751a6208168f.gif)


## Checklist
